src/framework/parsers/material/json-standard-material.js

```diff
@@ -132,7 +132,9 @@ class JsonStandardMaterialParser {
             ['diffuseMapTint', 'diffuseTint'],
             ['specularMapTint', 'specularTint'],
             ['emissiveMapTint', 'emissiveTint'],
-            ['metalnessMapTint', 'metalnessTint']
+            ['metalnessMapTint', 'metalnessTint'],
+
+            ['clearCoatGlossiness', 'clearCoatGloss']
```

Comment:
weren't there additional renamed gloss properties?

Reply:
no this was the only one